Sources/ComposableArchitecture/Internal/DispatchQueue.swift

```diff
@@ -12,6 +12,18 @@ func mainActorASAP(execute block: @escaping @MainActor @Sendable () -> Void) {
   }
 }
 
+func mainActorNow(execute block: @escaping @MainActor @Sendable () -> Void) {
```

Comment:
I added this helper to call from `TestStore.deinit`, since `deinit`s are currently non-isolated and so we can't otherwise invoke some of the teardown logic the test store performs otherwise.

This seems to work fine in practice, but if we're worried about deadlocks we could consider removing this helper and record an issue if `deinit` is ever called off the main thread instead.